test/blockchaintest/blockchaintest_loader.cpp

```diff
@@ -124,7 +115,17 @@ BlockchainTest load_blockchain_test_case(const std::string& name, const json::js
     bt.rev = to_rev_schedule(j.at("network").get<std::string>());
 
     for (const auto& el : j.at("blocks"))
-        bt.test_blocks.emplace_back(load_test_block(el, bt.rev.get_revision(0)));
+    {
+        if (const auto it = el.find("rlp_decoded"); it != el.end())
+        {
+            auto test_block = load_test_block(el.at("rlp_decoded"), bt.rev.get_revision(0));
```

Comment:
Do I get it right that "rlp_decoded" is some special scheme for invalid blocks? Please add short comment.

Reply:
yup, turns out it's even documented https://ethereum.github.io/execution-spec-tests/main/consuming_tests/blockchain_test/#invalidfixtureblock

Reply:
> So setting `test.block` to `false` is redundant in `load_test_block()` ?

Hm, good catch. Formally, it seems there can be no `rlp_decoded`. Then code here will read using `load_test_block(el, ...)`, and it will find the `exceptException` field there, and continue. It works correctly but not for the right reasons.

In practical terms now I see this is only the case for `eofwrap` fixtures, and per current release it actually makes tests fail if I omit this line :/.

Let me think a bit how to untie this knot easily. Worst case I have to go first and fix EEST's eofwrap, release there and bump here, and then that line in `load_test_block` should be redundant. We'll also not allow cases where `rlp_decoded` is not there, :crossed_fingers: there won't be any.

Reply:
okay, I think I cleaned this up. Code now reflects the structure of a blockchain test better. There is unfortunately a situation when there is no `rlp_decoded`, when the rlp representation of the block is invalid (duh), so we still skip those tests, but just a few (of 4844 tests at least).

The redundant check is removed. I've also submitted https://github.com/ethereum/execution-spec-tests/pull/1028 to EEST to fix eof wrap, until that is merged and I do a release there, CI will fail. Actually, unless I get a very quick feedback there, I'll point our CI to `ipsilon` "private" release to not stall this over the holidays.

Reply:
ooooh, right, those particular encoding tests are under 4844 so they are excluded, and CI is fine, so I think no obstacles to merge here. It is the followup #1077 which will have to wait.